### ISSUE TITLE
eslint gives validation error for variables used within bindings

### ISSUE THREAD

**Issue by bhavinkamani** [sveltejs/eslint-plugin-svelte3#85]
I am using intellij idea editor. I am also using the latest `3.1.0` release that supports typescript. 

With following code snippet eslint gives "Validation Error". It gives this error specifically for variables used within bindings. I get same errors while running eslint on cli.

![Screenshot 2021-02-17 at 12 36 47 PM](https://user-images.githubusercontent.com/120816/108168177-dc102280-711c-11eb-970c-6d3ef44ebae4.png)

Here's my configuration file.

```javascript
module.exports = {
    parserOptions: {
        ecmaVersion: 2021,
        sourceType: 'module',
        tsconfigRootDir: __dirname,
        project: ['./tsconfig.json'],
        extraFileExtensions: ['.svelte'],
    },
    parser: "@typescript-eslint/parser",
    env: {
        es6: true,
        browser: true,
        node: true,
    },
    extends: [
        "plugin:@typescript-eslint/recommended",
        'plugin:@typescript-eslint/recommended-requiring-type-checking',
    ],
    plugins: [
        'svelte3',
        "@typescript-eslint",
    ],
    overrides: [
        {
            files: ['*.svelte'],
            processor: 'svelte3/svelte3',
        }
    ],
    settings: {
        'svelte3/typescript': require('typescript'),
    },
};
```

**Comment by dummdidumm** [comment]
Should be fixed by #84

**Comment by Conduitry** [comment]
#84 has been released in 3.1.1.
